src/mongod_helpers.py

```diff
@@ -131,8 +131,6 @@ def primary_step_down(self) -> None:
             client.admin.command("replSetStepDown", config)
         except (ConnectionFailure, ConfigurationError, OperationFailure) as e:
             raise e
-        finally:
-            client.close()
```

Comment:
Is it correct to remove this connection close call? (the other calls to `client` are calling `close`)

Reply:
Not sure how this got removed, defintely necessary, great catch :)